### ISSUE TITLE
hillsborough_county_evictions_geocoded.csv formatting issue

### ISSUE THREAD

**Issue by dgillen** [datakind/Mar21-housing-insecurity#12]
Looks like there might be an issue with quoting vs delimiter issue  on a few rows?

![image](https://user-images.githubusercontent.com/4374331/110161453-fbf94480-7db2-11eb-8786-6a63b3874507.png)

**Comment by LeeBreisacher** [comment]
I'm looking at this, but I'm no expert, so if someone else knows more, feel free to go for it.

**Comment by snnehete** [comment]
It looks like the .csv is using doublequotes as an escape character for doublequotes inside fields. For instance, "11624 GROVEWOOD AVE ""B""" refers to the string '11624 GROVEWOOD AVE "B"'. That's confusing the read_csv() function because it reads anything in doublequotes as a string.

A regex to change the first doublequote to a more appropriate escape character, along with adding the escape character to the read_csv params, could work. I tried coming up with the regex myself but no luck so far.

**Comment by LeeBreisacher** [comment]
I noticed the original error message mentioned columns 61, 63, 65. I loaded the csv into a spreadsheet, scrolled to those columns, then scrolled down through the file looking for mistakes. I found one row that was "off" and noticed the match_indicator column contains just a small number of possible values. So I sorted on that column and quickly found about 6 rows that were wrong. Close examination of the input_addresss column revealed some extra/missing double quotes. I fixed those 6 rows by hand in a text editor.
